redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -121,24 +119,22 @@ internal class UIViewFlexContainer(
       return measure(size.useContents { toUnsafeSize() }).run { CGSizeMake(width, height) }
     }
 
-    override fun setNeedsLayout() {
-      needsLayout = true
```

Comment:
Adding a `super` call here fixed a bug, but `layoutSubviews` is only invoked when `setNeedsLayout` has been triggered so it turns out this can all be eliminated.